uber/reports.py

```diff
@@ -0,0 +1,48 @@
+from uber.common import *
+
+
```

Comment:
Literally anything can be one line of Python if you're insane enough:
https://github.com/csvoss/oneliner